main.go

```diff
@@ -28,8 +31,10 @@ import (
 )
 
 var setupLog = ctrl.Log.WithName("setup")
+var cpuprofile = flag.String("cpuprofile", "", "write cpu profile to file")
```

Comment:
A nitpick, but do we want to camelCase this as `cpuProfile`?

Reply:
Not as an arg to command line.....